server/lib/routes/get-index.js

```diff
@@ -2,6 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+var crypto = require('crypto');
+
+var activityEventQueryParams = [
+  'campaign',
```

Comment:
Thanks for catching that @vladikoff - @philbooth can you please also update the feature description to remove this field?  Using the `utm_*` params is a good idea but if it's too much mucking around, we can do it as follow-up work.